api/v1alpha1/hostdevicenetwork_types.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 const (
+	// HostDeviceNetworkCRDName is used for the CRD Kind.
```

Comment:
Most of the changes in this PR are adding comments or slightly changing existing comments to meet Go conventions.

Reply:
Enabling this now and fixing outstanding issues means we'll have a check for comments on packages and exported types in future.